sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
   message(STATUS "Download Level Zero loader and headers from github.com")
 
   set(LEVEL_ZERO_LOADER_REPO "https://github.com/oneapi-src/level-zero.git")
-  set(LEVEL_ZERO_LOADER_TAG v1.8.8)
+  set(LEVEL_ZERO_LOADER_TAG v1.11.0)
```

Comment:
Note for reviewers: the Level Zero loader version was increased because the previous version does not support implementation of [device IP version](https://github.com/oneapi-src/level-zero-spec/pull/23) Level Zero extension, required for implementation in SYCL

Reply:
> Note for reviewers: the Level Zero loader version was increased because the previous version does not support implementation of [device IP version](https://github.com/oneapi-src/level-zero-spec/pull/23) Level Zero extension, required for implementation in SYCL

Please, add this information to the PR description.

Reply:
Added to the description